src/app/static/src/app/store/baseline/mutations.ts

```diff
@@ -35,6 +35,8 @@ export const mutations: MutationTree<BaselineState> & BaselineMutations = {
     PJNZUpdated(state: BaselineState, action: PayloadWithType<PjnzResponse | null>) {
         const data = action.payload;
         if (data) {
+            console.log("----------------")
```

Comment:
You want this to stay in or was it just for testing?

Reply:
oops :) removed